File: sample/src/main/java/ru/terrakok/cicerone/sample/mvp/animation/profile/ProfileView.java
package ru.terrakok.cicerone.sample.mvp.animation.profile;

import com.arellomobile.mvp.MvpView;
import com.arellomobile.mvp.viewstate.strategy.AddToEndSingleStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

/**
 * Created by Konstantin Tskhovrebov (aka @terrakok) on 14.07.17.
 */

@StateStrategyType(AddToEndSingleStrategy.class)
public interface ProfileView extends MvpView {
    void showPhoto(int resId);
}

File: sample/src/main/java/com/github/terrakok/cicerone/sample/mvp/animation/profile/ProfileView.kt
package com.github.terrakok.cicerone.sample.mvp.animation.profile

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.StateStrategyType

/**
 * Created by Konstantin Tskhovrebov (aka @terrakok) on 14.07.17.
 */
@StateStrategyType(AddToEndSingleStrategy::class)
interface ProfileView : MvpView {
    fun showPhoto(resId: Int)
}